server/mscalendar/availability.go

```diff
@@ -171,15 +176,20 @@ func (m *mscalendar) setStatusFromAvailability(user *store.User, currentStatus s
 		if e.IsCancelled {
 			continue
 		}
-		h := fmt.Sprintf("%s %s", e.ID, e.Start.Time().UTC().Format(time.RFC3339))
+		h := fmt.Sprintf("%s %s", e.ICalUID, e.Start.Time().UTC().Format(time.RFC3339))
```

Comment:
Docs on `ICalID`: A unique identifier that is shared by all instances of an event across different calendars. Read-only.

I'm guessing the issues with the `ID` for a given event changing for whatever reason is solved by using `ICalID` instead. Do we want to replace all uses of `ID` in the codebase to avoid the issue elsewhere?

Reply:
Not in all places. We still need the ID to refer to the event when we talk with MSGraph. It should be used only on our side when we want to make sure it is the same event as other.